pkgs/development/go-modules/generic/default.nix

```diff
@@ -85,7 +85,12 @@ let
       runHook preBuild
 
       if [ ${deleteFlag} == "true" ]; then
-        rm -rf vendor
+        if [ ! -e vendor ]; then
+          echo "vendor folder does not exist, 'deleteVendor' is not needed"
```

Comment:
2/4 packages using `deleteVendor` don't have a vendor folder to delete. https://github.com/NixOS/nixpkgs/pull/95247